actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -504,6 +439,45 @@ def multiple_sources_tag_builder(type, sources)
           end
         end
 
+        def multiple_assets_tag_builder(type, sources, tag_name:, preload_as: nil, url_attr: "href")
+          options = sources.extract_options!.stringify_keys
+          path_options = options.extract!("protocol", "extname", "host", "skip_pipeline").symbolize_keys
+          options["crossorigin"] = "anonymous" if options["crossorigin"] == true
+          preload_links = []
+
+          yield options if block_given?
+
+          sources_tags = sources.uniq.map { |source|
+            url = send("path_to_#{type}", source, path_options)
+            if preload_as && preloadable_asset?(url, options["defer"])
+              preload_links << preload_header_for(url, preload_as, options)
+            end
+
+            tag(tag_name, { url_attr => url }.merge(options))
```

Comment:
i think this syntax is deprecated: https://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-tag

![image](https://user-images.githubusercontent.com/509837/135311185-a3727a17-d288-48cc-9174-76d6e8e49b8a.png)